api-go/internal/graph/graph.go

```diff
@@ -161,58 +159,58 @@ func SubGetSVG(w http.ResponseWriter, repo string, merge bool, charType string)
 // we need to also tell if the graph is ready to use on this side.
 // Try to get the endpoint of the line drawn and tell if it's on the right-most side
 func svgSucceed(svgBytes []byte) ([]byte, error) {
-	svg := string(svgBytes[:])
-	lines := strings.Split(svg, "\n")
-	var svgWidth float64
-	for _, l := range lines {
-		if strings.Contains(l, "<rect") {
-			words := strings.Split(l, " ")
-			for _, w := range words {
-				if strings.Contains(w, "width") {
-					parts := strings.Split(w, `"`)
-					var err error
-					svgWidth, err = strconv.ParseFloat(parts[1], 64)
-					if err != nil {
-						return nil, err
-					}
-					break
-				}
-			}
-		}
-	}
-	if svgWidth == 0 {
-		return nil, fmt.Errorf("could not get svg width")
-	}
-	lineColor := "39a85a"
-	for i, l := range lines {
-		if strings.Contains(l, lineColor) {
-			lineDrawn := strings.Split(strings.Split(l, `"`)[1], " ")
-			endPointX, err := strconv.ParseFloat(lineDrawn[len(lineDrawn)-2], 64)
-			if err != nil {
-				return nil, err
-			}
-			if float64(endPointX) < 0.95*float64(svgWidth) {
-				return nil, fmt.Errorf("the line is not reach its end")
-			}
-			break
-		}
-		if i == len(lines)-1 {
-			return nil, fmt.Errorf("could not get endpoint")
-		}
-	}
-	renderLengthMarker := "<path"
-	for i := len(lines) - 1; i >= 0; i-- {
-		if strings.Contains(lines[i], renderLengthMarker) {
-			words := strings.Split(lines[i], " ")
-			svgWidthStr := fmt.Sprintf("%f", svgWidth)
-			for j := range words {
-				if words[j] == "L" && j+1 < len(words) && words[j+1] != svgWidthStr {
-					lines[i] = strings.ReplaceAll(lines[i], words[j+1], svgWidthStr)
-					break
-				}
-			}
-			return []byte(strings.Join(lines, "\n")), nil
-		}
-	}
+	// svg := string(svgBytes[:])
```

Comment:
If we would like to remove this function, we could directly remove it and use git to look for history version